eng/pipelines/templates/jobs/tests-nightly-python.yml

```diff
@@ -18,11 +18,11 @@ jobs:
       - template: /eng/pipelines/templates/steps/use-python-version.yml
         parameters:
           versionSpec: $(PythonVersion)
-        
+
       - script: |
           python -m pip freeze
           python -m pip --version
-          python -m pip install setuptools==58.3.0 wheel==0.37.0 tox==4.5.0 packaging==23.1 requests
+          python -m pip install setuptools==72.2.0 wheel==0.37.0 tox==4.5.0 packaging==23.1 requests
           python -m pip install $(Build.SourcesDirectory)/tools/azure-sdk-tools[build]
```

Comment:
Do we still want to request a pinned version of `setuptools` before installing `azure-sdk-tools[build]` when `setuptools` (latest) is already a requirement? https://github.com/Azure/azure-sdk-for-python/blob/072f82316697960812ec1bd02fdf8bdf17011b54/tools/azure-sdk-tools/setup.py#L58

Reply:
yep. This is for consistency while invoking that pipeline. Because the extra requirement doesn't specify a required range, all it does is ensure it's installed. (which will will be)